.github/workflows/scripts/slack_gem_notifications/cve_methods.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'time'
+require 'feedjira'
+require 'httparty'
+
+def check_for_cves
+  xml = HTTParty.get('https://rubysec.com/atom.xml').body
```

Comment:
It would be good to have the RubySec feed URL live in a constant.

Reply:
Done ✅